tensorboard/examples/plugins/example_angular/README.md

```diff
@@ -0,0 +1,105 @@
+# Developing a TensorBoard plugin
```

Comment:
not sure it makes sense to duplicate the readme like this. Can we just move the one in example_basic to `examples/plugins/README.md` and add angular specific lines here?

Reply:
Yeah that's what I had in mind re the TODO in the description.  Sure, I can roll that change in here.